shuffle/app/api/polls/[id]/flaggedStatements/[statementId]/route.ts

```diff
@@ -0,0 +1,31 @@
+import { auth } from "@clerk/nextjs";
+import type { NextRequest } from "next/server";
+import { NextResponse } from "next/server";
+
+import prisma from "@/lib/prisma";
+import { requirePollAdminIfPollIsPrivate } from "@/utils/authutils";
+
+export async function DELETE(
```

Comment:
I thought we started using server actions. I guess it can stay here for now and we will refactor when it's ready for production.